Bottom: fb5d321b925392980f5d2671de1221db8f2f9446
Top:    1dee34cf2ff22055c37e30c6192cbcb0b53f04c0
Author: scott snyder <sss@karma>
Date:   2020-04-21 23:00:55 -0400

Remove references to GoComics strips that no longer exist.


---

diff --git a/ComicReader/assets/classes.json b/ComicReader/assets/classes.json
index 91a6049..dc810eb 100644
--- a/ComicReader/assets/classes.json
+++ b/ComicReader/assets/classes.json
@@ -134,7 +134,6 @@
     {"class":"GoComics.BergernWyse", "name":"Berger & Wyse", "pref":"bergerandwysePref", "new":"1"},
     {"class":"GoComics.Betty", "name":"Betty", "pref":"bettyPref"},
     {"class":"GoComics.Bewley", "name":"Bewley", "pref":"bewleyPref"},
-    {"class":"GoComics.BiffnRiley", "name":"Biff & Riley", "pref":"biffandrileyPref"},
     {"class":"GoComics.BigNate", "name":"Big Nate", "pref":"bignatePref"},
     {"class":"GoComics.TheBigPicture", "name":"The Big Picture", "pref":"thebigpicturePref"},
     {"class":"GoComics.BigTop", "name":"Big Top", "pref":"bigtopPref"},
@@ -155,7 +154,6 @@
     {"class":"GoComics.BroomHilda", "name":"Broom Hilda", "pref":"broomhildaPref"},
     {"class":"GoComics.TheBuckets", "name":"The Buckets", "pref":"thebucketsPref"},
     {"class":"GoComics.Buni", "name":"Buni", "pref":"buniPref"},
-    {"class":"GoComics.CafeconLeche", "name":"Cafe con Leche", "pref":"cafeconlechePref"},
     {"class":"GoComics.CalvinandHobbes", "name":"Calvin and Hobbes", "pref":"calvinandhobbesPref"},
     {"class":"GoComics.Candorville", "name":"Candorville", "pref":"candorvillePref"},
     {"class":"GoComics.Cathy", "name":"Cathy", "pref":"cathyPref"},
@@ -192,7 +190,6 @@
     {"class":"GoComics.Eek", "name":"Eek!", "pref":"eekPref"},
     {"class":"GoComics.TheElderberries", "name":"The Elderberries", "pref":"theelderberriesPref"},
     {"class":"GoComics.Endtown", "name":"Endtown", "pref":"endtownPref"},
-    {"class":"GoComics.ErictheCircle", "name":"Eric the Circle", "pref":"ericthecirclePref", "new":"1"},
     {"class":"GoComics.FMinus", "name":"F Minus", "pref":"fminusPref"},
     {"class":"GoComics.FamilyTree", "name":"Family Tree", "pref":"familytreePref"},
     {"class":"GoComics.Farcus", "name":"Farcus", "pref":"farcusPref"},
@@ -213,20 +210,16 @@
     {"class":"GoComics.FrogApplause", "name":"Frog Applause", "pref":"frogapplausePref"},
     {"class":"GoComics.TheFuscoBrothers", "name":"The Fusco Brothers", "pref":"thefuscobrothersPref"},
     {"class":"GoComics.Garfield", "name":"Garfield", "pref":"garfieldPref"},
-    {"class":"GoComics.GarfieldMinusGarfield", "name":"Garfield Minus Garfield", "pref":"garfieldminusgarfieldPref"},
     {"class":"GoComics.GasolineAlley", "name":"Gasoline Alley", "pref":"gasolinealleyPref"},
     {"class":"GoComics.Geech", "name":"Geech", "pref":"geechPref"},
     {"class":"GoComics.GetaLife", "name":"Get a Life", "pref":"getalifePref"},
     {"class":"GoComics.GetFuzzy", "name":"Get Fuzzy", "pref":"getfuzzyPref"},
     {"class":"GoComics.GilThorp", "name":"Gil Thorp", "pref":"gilthorpPref"},
     {"class":"GoComics.GingerMeggs", "name":"Ginger Meggs", "pref":"gingermeggsPref"},
-    {"class":"GoComics.GorDominical", "name":"Gor Dominical", "pref":"espanol/gordominicalPref"},
     {"class":"GoComics.Graffiti", "name":"Graffiti", "pref":"graffitiPref"},
     {"class":"GoComics.GrandAvenue", "name":"Grand Avenue", "pref":"grandavenuePref"},
     {"class":"GoComics.GrayMatters", "name":"Gray Matters", "pref":"graymattersPref", "new":"1"},
     {"class":"GoComics.TheGrizzwells", "name":"The Grizzwells", "pref":"thegrizzwellsPref"},
-    {"class":"GoComics.HaikuEwe", "name":"Haiku Ewe", "pref":"haikuewePref"},
-    {"class":"GoComics.HamShears", "name":"Ham Shears", "pref":"hamshearsPref"},
     {"class":"GoComics.HealthCapsules", "name":"Health Capsules", "pref":"healthcapsulesPref", "new":"1"},
     {"class":"GoComics.HeartoftheCity", "name":"Heart of the City", "pref":"heartofthecityPref"},
     {"class":"GoComics.Heathcliff", "name":"Heathcliff", "pref":"heathcliffPref"},
@@ -239,7 +232,6 @@
     {"class":"GoComics.ImagineThis", "name":"Imagine This", "pref":"imaginethisPref"},
     {"class":"GoComics.IntheBleachers", "name":"In the Bleachers", "pref":"inthebleachersPref"},
     {"class":"GoComics.IntheSticks", "name":"In the Sticks", "pref":"inthesticksPref"},
-    {"class":"GoComics.IncidentalComics", "name":"Incidental Comics", "pref":"incidentalcomicsPref"},
     {"class":"GoComics.InkPen", "name":"Ink Pen", "pref":"inkpenPref"},
     {"class":"GoComics.InspectorDangersCrimeQuiz", "name":"Inspector Danger's Crime Quiz", "pref":"inspectordangerscrimequizPref"},
     {"class":"GoComics.ItsAllAboutYou", "name":"It's All About You", "pref":"itsallaboutyouPref"},
@@ -248,7 +240,6 @@
     {"class":"GoComics.JoeVanilla", "name":"Joe Vanilla", "pref":"joevanillaPref"},
     {"class":"GoComics.JumpStart", "name":"Jump Start", "pref":"jumpstartPref"},
     {"class":"GoComics.TheKChronicles", "name":"The K Chronicles", "pref":"thekchroniclesPref"},
-    {"class":"GoComics.KidCity", "name":"KidCity", "pref":"kidcityPref"},
     {"class":"GoComics.KidSpot", "name":"KidSpot", "pref":"kidspotPref"},
     {"class":"GoComics.KitNCarlyle", "name":"Kit 'N' Carlyle", "pref":"kitandcarlylePref"},
     {"class":"GoComics.KitchenCapers", "name":"Kitchen Capers", "pref":"kitchencapersPref", "new":"1"},
@@ -258,7 +249,6 @@
     {"class":"GoComics.LaCucaracha", "name":"La Cucaracha", "pref":"lacucarachaPref"},
     {"class":"GoComics.LastKiss", "name":"Last Kiss", "pref":"lastkissPref"},
     {"class":"GoComics.TheLeftyBoscoPictureShow", "name":"The LeftyBosco Picture Show", "pref":"leftyboscopictureshowPref"},
-    {"class":"GoComics.LegendofBill", "name":"Legend of Bill", "pref":"legendofbillPref"},
     {"class":"GoComics.LibertyMeadows", "name":"Liberty Meadows", "pref":"libertymeadowsPref"},
     {"class":"GoComics.LilAbner", "name":"Li'l Abner", "pref":"lilabnerPref"},
     {"class":"GoComics.Lio", "name":"Lio", "pref":"lioPref"},
@@ -267,27 +257,21 @@
     {"class":"GoComics.LooseParts", "name":"Loose Parts", "pref":"loosepartsPref"},
     {"class":"GoComics.TheLostBear", "name":"The Lost Bear", "pref":"thelostbearPref"},
     {"class":"GoComics.LostSideofSuburbia", "name":"Lost Side of Suburbia", "pref":"lostsideofsuburbiaPref"},
-    {"class":"GoComics.LoveIs", "name":"Love Is...", "pref":"loveisPref"},
     {"class":"GoComics.Luann", "name":"Luann", "pref":"luannPref"},
     {"class":"GoComics.LuckyCow", "name":"Lucky Cow", "pref":"luckycowPref"},
-    {"class":"GoComics.Mac", "name":"Mac", "pref":"macPref"},
     {"class":"GoComics.MagicinaMinute", "name":"Magic in a Minute", "pref":"magicinaminutePref"},
     {"class":"GoComics.Maintaining", "name":"Maintaining", "pref":"maintainingPref"},
     {"class":"GoComics.MariasDay", "name":"Maria's Day", "pref":"mariasdayPref", "new":"1"},
     {"class":"GoComics.Marmaduke", "name":"Marmaduke", "pref":"marmadukePref"},
-    {"class":"GoComics.McArroni", "name":"McArroni", "pref":"mcarroniPref", "new":"1"},
     {"class":"GoComics.TheMeaningofLila", "name":"The Meaning of Lila", "pref":"meaningoflilaPref"},
     {"class":"GoComics.MediumLarge", "name":"Medium Large", "pref":"mediumlargePref", "new":"1"},
     {"class":"GoComics.MegClassics", "name":"Meg Classics", "pref":"megclassicsPref"},
     {"class":"GoComics.TheMiddletons", "name":"The Middletons", "pref":"themiddletonsPref"},
     {"class":"GoComics.MikeduJour", "name":"Mike du Jour", "pref":"mikedujourPref", "new":"1"},
-    {"class":"GoComics.MinimumSecurity", "name":"Minimum Security", "pref":"minimumsecurityPref"},
     {"class":"GoComics.ModeratelyConfused", "name":"Moderately Confused", "pref":"moderatelyconfusedPref"},
-    {"class":"GoComics.MollyandtheBear", "name":"Molly and the Bear", "pref":"mollyandthebearPref"},
     {"class":"GoComics.Momma", "name":"Momma", "pref":"mommaPref"},
     {"class":"GoComics.Monty", "name":"Monty", "pref":"montyPref"},
     {"class":"GoComics.MotleyClassics", "name":"Motley Classics", "pref":"motleyclassicsPref"},
-    {"class":"GoComics.MrGigiandtheSquid", "name":"Mr. Gigi and the Squid", "pref":"mrgigiandthesquidPref", "new":"1"},
     {"class":"GoComics.MuttnJeff", "name":"Mutt & Jeff", "pref":"muttandjeffPref"},
     {"class":"GoComics.MyCage", "name":"My Cage", "pref":"mycagePref"},
     {"class":"GoComics.MythTickle", "name":"MythTickle", "pref":"mythticklePref"},
@@ -307,10 +291,8 @@
     {"class":"GoComics.OutoftheGenePoolReRuns", "name":"Out of the Gene Pool Re-Runs", "pref":"outofthegenepoolPref"},
     {"class":"GoComics.OvertheHedge", "name":"Over the Hedge", "pref":"overthehedgePref"},
     {"class":"GoComics.Overboard", "name":"Overboard", "pref":"overboardPref"},
-    {"class":"GoComics.OysterWar", "name":"Oyster War", "pref":"oysterwarPref", "new":"1"},
     {"class":"GoComics.PCandPixel", "name":"PC and Pixel", "pref":"pcandpixelPref"},
     {"class":"GoComics.Peanuts", "name":"Peanuts", "pref":"peanutsPref"},
-    {"class":"GoComics.PeanutsHolidayCountdown", "name":"Peanuts Holiday Countdown", "pref":"peanutsholidaycountdownPref", "new":"1"},
     {"class":"GoComics.PearlsBeforeSwine", "name":"Pearls Before Swine", "pref":"pearlsbeforeswinePref"},
     {"class":"GoComics.Pibgorn", "name":"Pibgorn", "pref":"pibgornPref"},
     {"class":"GoComics.PibgornSketches", "name":"Pibgorn Sketches", "pref":"pibgornsketchesPref"},
@@ -328,7 +310,6 @@
     {"class":"GoComics.RedMeat", "name":"Red Meat", "pref":"redmeatPref"},
     {"class":"GoComics.ReplyAll", "name":"Reply All", "pref":"replyallPref"},
     {"class":"GoComics.RichardsPoorAlmanac", "name":"Richard's Poor Almanac", "pref":"richardspooralmanacPref", "new":"1"},
-    {"class":"GoComics.RicigsToonTrivia", "name":"Ricig's Toon Trivia", "pref":"ricigstoontriviaPref", "new":"1"},
     {"class":"GoComics.RipHaywire", "name":"Rip Haywire", "pref":"riphaywirePref"},
     {"class":"GoComics.RipleysBelieveItorNot", "name":"Ripley's Believe It or Not", "pref":"ripleysbelieveitornotPref"},
     {"class":"GoComics.RoseisRose", "name":"Rose is Rose", "pref":"roseisrosePref"},
@@ -343,10 +324,8 @@
     {"class":"GoComics.SkinHorse", "name":"Skin Horse", "pref":"skinhorsePref"},
     {"class":"GoComics.Skippy", "name":"Skippy", "pref":"skippyPref", "new":"1"},
     {"class":"GoComics.Slowpoke", "name":"Slowpoke", "pref":"slowpokePref"},
-    {"class":"GoComics.SoupToNutz", "name":"Soup To Nutz", "pref":"souptonutzPref"},
     {"class":"GoComics.SpeedBump", "name":"Speed Bump", "pref":"speedbumpPref"},
     {"class":"GoComics.SpottheFrog", "name":"Spot the Frog", "pref":"spotthefrogPref"},
-    {"class":"GoComics.Starslip", "name":"Starslip", "pref":"starslipPref", "new":"1"},
     {"class":"GoComics.StoneSoup", "name":"Stone Soup", "pref":"stonesoupPref"},
     {"class":"GoComics.StrangeBrew", "name":"Strange Brew", "pref":"strangebrewPref"},
     {"class":"GoComics.TheSunshineClub", "name":"The Sunshine Club", "pref":"thesunshineclubPref"},
@@ -354,21 +333,17 @@
     {"class":"GoComics.TankMcNamara", "name":"Tank McNamara", "pref":"tankmcnamaraPref"},
     {"class":"GoComics.Tarzan", "name":"Tarzan", "pref":"tarzanPref", "new":"1"},
     {"class":"GoComics.TenCats", "name":"Ten Cats", "pref":"tencatsPref"},
-    {"class":"GoComics.TalesofTerraTopia", "name":"Tales of TerraTopia", "pref":"terratopiaPref"},
     {"class":"GoComics.ThatisPriceless", "name":"That is Priceless", "pref":"thatispricelessPref"},
     {"class":"GoComics.ThatsLife", "name":"That's Life", "pref":"thatslifePref"},
     {"class":"GoComics.Thatababy", "name":"Thatababy", "pref":"thatababyPref"},
     {"class":"GoComics.ThinLines", "name":"Thin Lines", "pref":"thinlinesPref"},
     {"class":"GoComics.TinySepuku", "name":"Tiny Sepuku", "pref":"tinysepukuPref"},
     {"class":"GoComics.Toby", "name":"Toby", "pref":"tobyPref"},
-    {"class":"GoComics.TodaysDogg", "name":"Today's Dogg", "pref":"todaysdoggPref"},
     {"class":"GoComics.TomtheDancingBug", "name":"Tom the Dancing Bug", "pref":"tomthedancingbugPref"},
     {"class":"GoComics.TooMuchCoffeeMan", "name":"Too Much Coffee Man", "pref":"toomuchcoffeemanPref"},
     {"class":"GoComics.Trivquiz", "name":"Trivquiz", "pref":"trivquizPref", "new":"1"},
-    {"class":"GoComics.Twaggies", "name":"Twaggies", "pref":"twaggiesPref"},
     {"class":"GoComics.UncleArtsFunland", "name":"Uncle Art's Funland", "pref":"uncleartsfunlandPref"},
     {"class":"GoComics.UnstrangePhenomena", "name":"Unstrange Phenomena", "pref":"unstrangephenomenaPref"},
-    {"class":"GoComics.USAcres", "name":"U.S. Acres", "pref":"usacresPref"},
     {"class":"GoComics.ViivinWagner", "name":"Viivi & Wagner", "pref":"viiviandwagnerPref", "new":"1"},
     {"class":"GoComics.WatchYourHead", "name":"Watch Your Head", "pref":"watchyourheadPref"},
     {"class":"GoComics.WeePals", "name":"Wee Pals", "pref":"weepalsPref"},
@@ -384,8 +359,6 @@
     {"class":"GoComics.LaloAlcaraz", "name":"Editorial : Lalo Alcaraz", "pref":"laloalcarazPref"},
     {"class":"GoComics.NickAnderson", "name":"Editorial : Nick Anderson", "pref":"nickandersonPref"},
     {"class":"GoComics.RobertAriail", "name":"Editorial : Robert Ariail", "pref":"robertariailPref"},
-    {"class":"GoComics.ChuckAsay", "name":"Editorial : Chuck Asay", "pref":"chuckasayPref"},
-    {"class":"GoComics.TonyAuth", "name":"Editorial : Tony Auth", "pref":"tonyauthPref"},
     {"class":"GoComics.ClayBennett", "name":"Editorial : Clay Bennett", "pref":"claybennettPref"},
     {"class":"GoComics.LisaBenson", "name":"Editorial : Lisa Benson", "pref":"lisabensonPref"},
     {"class":"GoComics.SteveBenson", "name":"Editorial : Steve Benson", "pref":"stevebensonPref"},
@@ -401,17 +374,13 @@
     {"class":"GoComics.BobGorrell", "name":"Editorial : Bob Gorrell", "pref":"bobgorrellPref"},
     {"class":"GoComics.WaltHandelsman", "name":"Editorial : Walt Handelsman", "pref":"walthandelsmanPref"},
     {"class":"GoComics.JoeHeller", "name":"Editorial : Joe Heller", "pref":"joehellerPref", "new":"1"},
-    {"class":"GoComics.JerryHolbert", "name":"Editorial : Jerry Holbert", "pref":"jerryholbertPref"},
     {"class":"GoComics.ClayJones", "name":"Editorial : Clay Jones", "pref":"clayjonesPref"},
     {"class":"GoComics.KevinKallaugher", "name":"Editorial : Kevin Kallaugher", "pref":"kevinkallaugherPref"},
     {"class":"GoComics.SteveKelley", "name":"Editorial : Steve Kelley", "pref":"stevekelleyPref"},
     {"class":"GoComics.MikeLester", "name":"Editorial : Mike Lester", "pref":"mikelesterPref"},
     {"class":"GoComics.DrewLitton", "name":"Editorial : Drew Litton", "pref":"drewlittonPref"},
-    {"class":"GoComics.DickLocher", "name":"Editorial : Dick Locher", "pref":"dicklocherPref"},
-    {"class":"GoComics.ChanLowe", "name":"Editorial : Chan Lowe", "pref":"chanlowePref"},
     {"class":"GoComics.MikeLuckovich", "name":"Editorial : Mike Luckovich", "pref":"mikeluckovichPref"},
     {"class":"GoComics.GaryMarkstein", "name":"Editorial : Gary Markstein", "pref":"garymarksteinPref"},
-    {"class":"GoComics.GlennMcCoy", "name":"Editorial : Glenn McCoy", "pref":"glennmccoyPref"},
     {"class":"GoComics.JimMorin", "name":"Editorial : Jim Morin", "pref":"jimmorinPref"},
     {"class":"GoComics.JackOhman", "name":"Editorial : Jack Ohman", "pref":"jackohmanPref"},
     {"class":"GoComics.PatOliphant", "name":"Editorial : Pat Oliphant", "pref":"patoliphantPref"},
@@ -421,16 +390,12 @@
     {"class":"GoComics.MichaelRamirez", "name":"Editorial : Michael Ramirez", "pref":"michaelramirezPref"},
     {"class":"GoComics.MarshallRamsey", "name":"Editorial : Marshall Ramsey", "pref":"marshallramseyPref"},
     {"class":"GoComics.RobRogers", "name":"Editorial : Rob Rogers", "pref":"robrogersPref"},
-    {"class":"GoComics.SteveSack", "name":"Editorial : Steve Sack", "pref":"stevesackPref"},
-    {"class":"GoComics.BenSargent", "name":"Editorial : Ben Sargent", "pref":"bensargentPref"},
     {"class":"GoComics.DrewSheneman", "name":"Editorial : Drew Sheneman", "pref":"drewshenemanPref"},
-    {"class":"GoComics.SmallWorld", "name":"Editorial : Small World", "pref":"smallworldPref"},
     {"class":"GoComics.JeffStahler", "name":"Editorial : Jeff Stahler", "pref":"jeffstahlerPref"},
     {"class":"GoComics.ScottStantis", "name":"Editorial : Scott Stantis", "pref":"scottstantisPref"},
     {"class":"GoComics.DanaSummers", "name":"Editorial : Dana Summers", "pref":"danasummersPref"},
     {"class":"GoComics.PaulSzep", "name":"Editorial : Paul Szep", "pref":"paulszepPref"},
     {"class":"GoComics.Think", "name":"Editorial : (th)ink", "pref":"thinkPref"},
-    {"class":"GoComics.MikeThompson", "name":"Editorial : Mike Thompson", "pref":"mikethompsonPref"},
     {"class":"GoComics.TomToles", "name":"Editorial : Tom Toles", "pref":"tomtolesPref"},
     {"class":"GoComics.GaryVarvel", "name":"Editorial : Gary Varvel", "pref":"garyvarvelPref"},
     {"class":"GoComics.ViewsAfrica", "name":"Editorial : ViewsAfrica", "pref":"viewsafricaPref"},
@@ -444,7 +409,6 @@
     {"class":"GoComics.CaricaturesbyKerryWaghorn", "name":"Editorial : Caricatures by Kerry Waghorn", "pref":"facesinthenewsPref"},
     {"class":"GoComics.DanWasserman", "name":"Editorial : Dan Wasserman", "pref":"danwassermanPref"},
     {"class":"GoComics.SigneWilkinson", "name":"Editorial : Signe Wilkinson", "pref":"signewilkinsonPref"},
-    {"class":"GoComics.WitoftheWorld", "name":"Editorial : Wit of the World", "pref":"witoftheworldPref"},
     {"class":"GoComics.MattWuerker", "name":"Editorial : Matt Wuerker", "pref":"mattwuerkerPref"}
   ]
 }
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java
index 524b305..ee73ec2 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java
@@ -89,6 +89,7 @@ public class CtrlAltDel extends DailyComic {
   
         protected Calendar getTimeFromUrl(String url)
         {
+          Log.d ("zzz", url);
           int i = url.lastIndexOf ("ENG_");
           if (i >= 0) {
             i += 4;
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/BenSargent.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/BenSargent.java
deleted file mode 100755
index 378ea09..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/BenSargent.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class BenSargent extends DailyGoComicsCom {
-	public BenSargent() {
-		super();
-		mComicName = "bensargent";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(1999, 6, 1);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/BiffnRiley.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/BiffnRiley.java
deleted file mode 100755
index 877a41e..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/BiffnRiley.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class BiffnRiley extends DailyGoComicsCom {
-	public BiffnRiley() {
-		super();
-		mComicName = "biff-and-riley";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2010, 0, 12);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/CafeconLeche.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/CafeconLeche.java
deleted file mode 100755
index 530c4d4..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/CafeconLeche.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class CafeconLeche extends DailyGoComicsCom {
-	public CafeconLeche() {
-		super();
-		mComicName = "cafeconleche";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2010, 4, 7);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ChanLowe.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ChanLowe.java
deleted file mode 100755
index eb4a726..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ChanLowe.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class ChanLowe extends DailyGoComicsCom {
-	public ChanLowe() {
-		super();
-		mComicName = "chanlowe";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2001, 3, 23);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ChuckAsay.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ChuckAsay.java
deleted file mode 100755
index 955fc47..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ChuckAsay.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class ChuckAsay extends DailyGoComicsCom {
-	public ChuckAsay() {
-		super();
-		mComicName = "chuckasay";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2002, 7, 4);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/DickLocher.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/DickLocher.java
deleted file mode 100755
index d467f80..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/DickLocher.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class DickLocher extends DailyGoComicsCom {
-	public DickLocher() {
-		super();
-		mComicName = "dicklocher";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2001, 3, 20);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ErictheCircle.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ErictheCircle.java
deleted file mode 100755
index adbc4a9..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/ErictheCircle.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class ErictheCircle extends DailyGoComicsCom {
-	public ErictheCircle() {
-		super();
-		mComicName = "eric-the-circle";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 10, 1);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GarfieldMinusGarfield.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GarfieldMinusGarfield.java
deleted file mode 100755
index b77e127..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GarfieldMinusGarfield.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class GarfieldMinusGarfield extends DailyGoComicsCom {
-	public GarfieldMinusGarfield() {
-		super();
-		mComicName = "garfieldminusgarfield";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2008, 10, 3);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GlennMcCoy.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GlennMcCoy.java
deleted file mode 100755
index 34e9800..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GlennMcCoy.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class GlennMcCoy extends DailyGoComicsCom {
-	public GlennMcCoy() {
-		super();
-		mComicName = "glennmccoy";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(1999, 7, 25);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GorDominical.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GorDominical.java
deleted file mode 100755
index 13d1127..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/GorDominical.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class GorDominical extends DailyGoComicsCom {
-	public GorDominical() {
-		super();
-		mComicName = "espanol/gor-dominical";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 4, 22);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/HaikuEwe.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/HaikuEwe.java
deleted file mode 100755
index 98b9a65..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/HaikuEwe.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class HaikuEwe extends DailyGoComicsCom {
-	public HaikuEwe() {
-		super();
-		mComicName = "haikuewe";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2009, 2, 9);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/HamShears.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/HamShears.java
deleted file mode 100755
index 9983523..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/HamShears.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class HamShears extends DailyGoComicsCom {
-	public HamShears() {
-		super();
-		mComicName = "ham-shears";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 10, 7);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/IncidentalComics.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/IncidentalComics.java
deleted file mode 100755
index 6b1c353..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/IncidentalComics.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class IncidentalComics extends DailyGoComicsCom {
-	public IncidentalComics() {
-		super();
-		mComicName = "incidentalcomics";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 2, 21);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/JerryHolbert.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/JerryHolbert.java
deleted file mode 100755
index b453375..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/JerryHolbert.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class JerryHolbert extends DailyGoComicsCom {
-	public JerryHolbert() {
-		super();
-		mComicName = "jerryholbert";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 5, 1);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/KidCity.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/KidCity.java
deleted file mode 100755
index 20e0bb0..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/KidCity.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class KidCity extends DailyGoComicsCom {
-	public KidCity() {
-		super();
-		mComicName = "kidcity";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 5, 5);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/LegendofBill.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/LegendofBill.java
deleted file mode 100755
index eb6e260..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/LegendofBill.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class LegendofBill extends DailyGoComicsCom {
-	public LegendofBill() {
-		super();
-		mComicName = "legendofbill";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2008, 4, 27);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/LoveIs.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/LoveIs.java
deleted file mode 100755
index d965154..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/LoveIs.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class LoveIs extends DailyGoComicsCom {
-	public LoveIs() {
-		super();
-		mComicName = "loveis";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2007, 11, 17);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Mac.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Mac.java
deleted file mode 100755
index 9e9d309..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Mac.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class Mac extends DailyGoComicsCom {
-	public Mac() {
-		super();
-		mComicName = "mac";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2012, 4, 20);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/McArroni.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/McArroni.java
deleted file mode 100755
index 51fbed4..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/McArroni.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class McArroni extends DailyGoComicsCom {
-	public McArroni() {
-		super();
-		mComicName = "mcarroni";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2012, 9, 24);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MikeThompson.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MikeThompson.java
deleted file mode 100755
index 84b7047..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MikeThompson.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class MikeThompson extends DailyGoComicsCom {
-	public MikeThompson() {
-		super();
-		mComicName = "mikethompson";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2008, 3, 8);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MinimumSecurity.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MinimumSecurity.java
deleted file mode 100755
index 6c01277..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MinimumSecurity.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class MinimumSecurity extends DailyGoComicsCom {
-	public MinimumSecurity() {
-		super();
-		mComicName = "minimumsecurity";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2001, 8, 3);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MollyandtheBear.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MollyandtheBear.java
deleted file mode 100755
index 8bd6008..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MollyandtheBear.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class MollyandtheBear extends DailyGoComicsCom {
-	public MollyandtheBear() {
-		super();
-		mComicName = "mollyandthebear";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2009, 0, 19);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MrGigiandtheSquid.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MrGigiandtheSquid.java
deleted file mode 100755
index 9fbc444..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/MrGigiandtheSquid.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class MrGigiandtheSquid extends DailyGoComicsCom {
-	public MrGigiandtheSquid() {
-		super();
-		mComicName = "mr-gigi-and-the-squid";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2012, 6, 11);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/OysterWar.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/OysterWar.java
deleted file mode 100755
index fd7c920..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/OysterWar.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class OysterWar extends DailyGoComicsCom {
-	public OysterWar() {
-		super();
-		mComicName = "oyster-war";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2012, 10, 14);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/PeanutsHolidayCountdown.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/PeanutsHolidayCountdown.java
deleted file mode 100755
index ebe7c26..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/PeanutsHolidayCountdown.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class PeanutsHolidayCountdown extends DailyGoComicsCom {
-	public PeanutsHolidayCountdown() {
-		super();
-		mComicName = "peanuts-holiday-countdown";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 10, 27);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/RicigsToonTrivia.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/RicigsToonTrivia.java
deleted file mode 100755
index 1dbd63e..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/RicigsToonTrivia.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class RicigsToonTrivia extends DailyGoComicsCom {
-	public RicigsToonTrivia() {
-		super();
-		mComicName = "ricigs-toon-trivia";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2012, 11, 4);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SmallWorld.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SmallWorld.java
deleted file mode 100755
index ab40c54..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SmallWorld.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class SmallWorld extends DailyGoComicsCom {
-	public SmallWorld() {
-		super();
-		mComicName = "smallworld";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2003, 3, 20);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SoupToNutz.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SoupToNutz.java
deleted file mode 100755
index fd699eb..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SoupToNutz.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class SoupToNutz extends DailyGoComicsCom {
-	public SoupToNutz() {
-		super();
-		mComicName = "soup-to-nutz";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 4, 16);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Starslip.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Starslip.java
deleted file mode 100755
index e96eb0c..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Starslip.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class Starslip extends DailyGoComicsCom {
-	public Starslip() {
-		super();
-		mComicName = "starslip";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2012, 6, 6);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SteveSack.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SteveSack.java
deleted file mode 100755
index 9664f71..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/SteveSack.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class SteveSack extends DailyGoComicsCom {
-	public SteveSack() {
-		super();
-		mComicName = "stevesack";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2001, 3, 19);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TalesofTerraTopia.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TalesofTerraTopia.java
deleted file mode 100755
index 2afc9af..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TalesofTerraTopia.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class TalesofTerraTopia extends DailyGoComicsCom {
-	public TalesofTerraTopia() {
-		super();
-		mComicName = "terratopia";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 9, 17);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TodaysDogg.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TodaysDogg.java
deleted file mode 100755
index 1521703..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TodaysDogg.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class TodaysDogg extends DailyGoComicsCom {
-	public TodaysDogg() {
-		super();
-		mComicName = "todays-dogg";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 4, 31);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TonyAuth.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TonyAuth.java
deleted file mode 100755
index 667eeaf..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/TonyAuth.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class TonyAuth extends DailyGoComicsCom {
-	public TonyAuth() {
-		super();
-		mComicName = "tonyauth";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(1996, 3, 5);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Twaggies.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Twaggies.java
deleted file mode 100755
index 5dedcf1..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/Twaggies.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class Twaggies extends DailyGoComicsCom {
-	public Twaggies() {
-		super();
-		mComicName = "twaggies";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2011, 11, 13);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/USAcres.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/USAcres.java
deleted file mode 100755
index 602993f..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/USAcres.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class USAcres extends DailyGoComicsCom {
-	public USAcres() {
-		super();
-		mComicName = "us-acres";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2012, 2, 3);
-	}
-}
diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/WitoftheWorld.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/WitoftheWorld.java
deleted file mode 100755
index 2ed025b..0000000
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GoComics/WitoftheWorld.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package com.blogspot.applications4android.comicreader.comics.GoComics;
-
-import java.util.Calendar;
-import com.blogspot.applications4android.comicreader.comictypes.DailyGoComicsCom;
-
-public class WitoftheWorld extends DailyGoComicsCom {
-	public WitoftheWorld() {
-		super();
-		mComicName = "witoftheworld";
-		mFirstCal = Calendar.getInstance();
-		mFirstCal.set(2006, 11, 24);
-	}
-}
